thegame/engine/game_objects/character_object.py

```diff
@@ -16,3 +17,61 @@ def player_interaction(self, keystrokes, context):
             tell if w was pressed, and if it is, run a method
             to move the character forward."""
         raise NotImplementedError()
+
+
+class PlayerCharacter(PlayerControlledObject):
+    """ An ease-of-use class that embodies a PC.
+        This class is a PlayerControlledObject which
+        has built in functionality for movement."""
+
+    def player_interaction(self, keystrokes, context):
+
+        # Check if any of WASD were pressed, and if so move.
+        if (
+            "w" in keystrokes
+            or "s" in keystrokes
+            or "a" in keystrokes
+            or "d" in keystrokes
+        ):
+            self.move(
+                context,
+                up="w" in keystrokes,
+                down="s" in keystrokes,
+                left="a" in keystrokes,
+                right="d" in keystrokes,
+            )
+
+    def move(self, context, up=False, down=False, left=False, right=False):
+        """ A class that moves the PC around the map in the directions specified.
+            Only one direction is supported at a time.
+            Note, going up always takes priority, going up or down takes priority
+            over going left and right, and going left takes priority over going right.
+            This means that if up and down, then the PC will move up; if up and
+            left, then the PC will move up; if down and right, then the PC will
+            move down; and if left and right, the PC will move left."""
+
+        current_pos = context.player_controlled_objects[self]
+        sheet = context.active_screen.CHARACTER_SHEET_INDEX
+        logging.info(
+            f"PlayerControlledObject at {current_pos} is moving with {(up, down, left, right)}. "
```

Comment:
Don't need the current value anymore, and this should probably be moved to debug level logging.